docs/docs/20-usage/33-continuous-deployment.md

```diff
@@ -0,0 +1,41 @@
+# Continuous Deployment
```

Comment:
In general I would like this is a nice overview, but I think its to generic to be actually helpful. I think we could rather add a new category: cookbook #316 or articles to our  new blog with concrete tutorial styled guides. So something like to deploy a docker container with ansible you would first build the container with WP, add it to a registry and then run an ansible script to deploy the new container.